environment-dev.yml

```diff
@@ -77,7 +77,7 @@ dependencies:
   - pyinstrument
   - memray
   - pip:
-      - sherpa
+      - sherpa>=4.17
```

Comment:
```suggestion
      - sherpa<=4.17
```

Reply:
I am not sure about that... The dev should handle new versions! 
The same type of modifications in the other subsections have been merged, but I am not sure about it.